rsyslog/Sanity/capabilities-drop/runtest.sh

```diff
@@ -46,7 +46,7 @@ rlJournalStart && {
     rsyslogConfigAddTo "GLOBALS" < <(rsyslogConfigCreateSection 'CAP')
   rlPhaseEnd; }
 
-  exp_caps="block_suspend chown ipc_lock lease net_admin net_bind_service perfmon setgid setuid sys_admin sys_chroot syslog sys_resource"
+  exp_caps="block_suspend chown dac_override ipc_lock lease net_admin net_bind_service perfmon setgid setuid sys_admin sys_chroot syslog sys_resource"
```

Comment:
Recently we discovered that the CAP_PERFMON capability is not needed and it was removed from upstream as well as part of the original PR. Defined capabilities: https://github.com/rsyslog/rsyslog/blob/master/tools/rsyslogd.c#L2181

Reply:
Is that written somewhere? Like can I base the change on some particular issue?

Reply:
The initial list was mentioned at https://github.com/rsyslog/rsyslog/pull/4999#issuecomment-1313362425 .

Reply:
Ok, the list was created based on some of the actual rsyslog builds.